index.js

```diff
@@ -246,4 +246,59 @@ Paper.prototype.render = function (path, context) {
     return output;
 };
 
+/**
+ * Theme rendering logic
+ * @param  {String|Array} templatePath
+ * @param  {Object} data
+ * @return {String|Object}
+ */
+Paper.prototype.renderTheme = function(templatePath, data) {
+    var self = this,
+        html,
+        output;
+
+    data.context.template_file = templatePath;
+
+    // Is an ajax request?
+    if (data.remote || _.isArray(templatePath)) {
+
+        if (data.remote) {
+            data.context = _.extend({}, data.context, data.remote_data);
+        }
+
+        // Is render_with ajax request?
+        if (templatePath) {
+            // if multiple render_with
+            if (_.isArray(templatePath)) {
+                // if templatePath is an array ( multiple templates using render_with option)
+                // compile all the template required files into a hash table
+                html = templatePath.reduce(function(table, file) {
```

Comment:
switch yo arrow function syntax, by doing this you can use `this` and remove the `self` variable